examples/scripts/get-envconf.py

```diff
@@ -0,0 +1,127 @@
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in
+# all copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+# THE SOFTWARE.
+###
+from __future__ import print_function
+from __future__ import unicode_literals
+from __future__ import division
+from __future__ import absolute_import
+from builtins import range
+from future import standard_library
+standard_library.install_aliases()
+import sys
+
+PYTHON_VERSION = sys.version_info[:3]
+PY2 = (PYTHON_VERSION[0] == 2)
+if PY2:
+    if PYTHON_VERSION < (2, 7, 9):
+        raise Exception('Must use Python 2.7.9 or later')
+elif PYTHON_VERSION < (3, 4):
+    raise Exception('Must use Python 3.4 or later')
+
+from pprint import pprint
+import hpOneView as hpov
+
+
+def acceptEULA(con):
+    # See if we need to accept the EULA before we try to log in
+    con.get_eula_status()
+    try:
+        if con.get_eula_status() is True:
+            print('EULA display needed')
+            con.set_eula('no')
+    except Exception as e:
+        print('EXCEPTION:')
+        print(e)
+
+
+def login(con, credential):
+    # Login with givin credentials
+    try:
+        con.login(credential)
+    except:
+        print('Login failed')
+
+
+def getenvconf(srv, server):
+    for s in srv.get_servers():
+        if s['name'] == server:
+            ret = srv.get_env_conf(s)
+        else:
+            ret = 'This server does not exist.'
+    pprint(ret)
+
+
+def main():
+    parser = argparse.ArgumentParser(add_help=True,
+                        formatter_class=argparse.RawTextHelpFormatter,
+                                     description='''
+    Display Server Resources
+
+    Usage: ''')
+    parser.add_argument('-a', dest='host', required=True,
+                        help='''
+    HP OneView Appliance hostname or IP address''')
+    parser.add_argument('-u', dest='user', required=False,
+                        default='Administrator',
+                        help='''
+    HP OneView Username''')
+    parser.add_argument('-p', dest='passwd', required=True,
+                        help='''
+    HP OneView Password''')
+    parser.add_argument('-n', dest='name',
+                        required=False,
+                        help='''
+    The name of the server hardware resource to be returned its utilization.
```

Comment:
should line 93 describe that the script is returning environmental configuration